chapter02/docker-compose.yml

```diff
@@ -0,0 +1,306 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@ismaelcv For starting this airflow the chapter mentions using the shell script. Docker compose i use form chapter 3 on. I'm not against having the compose file here but it should be a deliberate choice.

Reply:
For development purposes i would like to include it. 
Its really handy to have a compose file there to be ready to run the examples

Reply:
Fine with me.